pkg/bundle/bundle.go

```diff
@@ -35,6 +35,7 @@ import (
 
 const SigstoreBundleMediaType01 = "application/vnd.dev.sigstore.bundle+json;version=0.1"
 const SigstoreBundleMediaType02 = "application/vnd.dev.sigstore.bundle+json;version=0.2"
+const SigstoreBundleMediaType03 = "application/vnd.dev.sigstore.bundle.v0.3+json"
```

Comment:
It looks like we're going to support the old v3 string still, so even though I don't think there are any bundles floating about, can we check both formats?

Reply:
I added the legacy string as well